_posts/eakita/2020-08-26-logical_turtle-eakita.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+author: eakita
```

Comment:
In the examples author comes first, above layout- not sure if this will make a difference.